infra/cifuzz/cifuzz_test.py

```diff
@@ -114,60 +122,53 @@ def test_invalid_workspace(self):
 class RunFuzzersIntegrationTest(unittest.TestCase):
   """Test build_fuzzers function in the cifuzz module."""
 
+  def tearDown(self):
+    """Remove any existing crashes and test files."""
+    out_dir = os.path.join(TEST_FILES_PATH, 'out')
+    for out_file in os.listdir(out_dir):
+      out_path = os.path.join(out_dir, out_file)
+      if out_file != EXAMPLE_FUZZER:
```

Comment:
```
if out_file == EXAMPLE_FUZZER:
  continue
```

and the rest of the code below can have a smaller indentation

Reply:
Added.